src/hooks/samples/UseCustomQuery.jsx

```diff
@@ -7,8 +7,9 @@ import { setLanguage, setSearchEndpoint } from '../../services/api/endpoints';
 const UseCustomQuerySample = () => {
 	const apiBaseEndpoint = 'http://localhost:3000/api';
 	const lang = 'en';
-	setLanguage(lang);
-	setSearchEndpoint(`${apiBaseEndpoint}/glossary/v1/search`);
+  setLanguage(lang);
+  console.log(`${apiBaseEndpoint}/sitewidesearch/v1/Search/cgov/${lang}`);
```

Comment:
Was this console.log a part of debugging and left here?

Reply:
Yup, that was probably an omission.